src/NEO/Constants.h

```diff
@@ -0,0 +1,16 @@
+// Copyright © 2017-2022 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#pragma once
+
+namespace TW::NEO {
+
+static const size_t assetIdSize = 32;
+static const size_t contractHashSize = 32;
+static const size_t valueSize = 8;
+static const size_t scriptHashSize = 20;
+
+} // namespace TW::NEO
```

Comment:
can you add an empty line to the end of each file changed?

Reply:
done